### PR TITLE
ref(js): Cleanup SharedGroupDetails

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Comment by scttcper** [comment]
looks like mobile is overflowing

File: static/app/views/sharedGroupDetails/index.tsx [hunk 1]
```diff
@@ -103,45 +95,47 @@ class SharedGroupDetails extends Component<Props, State> {
...
+    const {title, permalink, latestEvent, project} = group;
+
+    // XXX(epurkhiser): Be careful. The organization represented here as part
+    // of the shared group's project is NOT a full organization, just a slug
+    // and name.
+    const organization: SharedViewOrganization = project.organization;
 
     return (
-      <DocumentTitle title={title}>
+      <DocumentTitle title={title ?? t('Sentry')}>
```
Comment (priscilawebdev) [inline static/app/views/sharedGroupDetails/index.tsx:106]:
```suggestion
      <DocumentTitle title={title ?? Sentry}>
```

**Comment by priscilawebdev** [comment]
should we fix the 'Details' link style?

Before:
image

After:
image

**Comment by priscilawebdev** [comment]
found another issue.

Before:

image


After:

image

**Pull request closed without merging**